packages/terra-status-view/src/StatusView.jsx

```diff
@@ -99,7 +99,7 @@ const StatusView = ({
   } else if (variant && !isGlyphHidden) {
     glyphSection = (
       <div className={cx('glyph')}>
-        <svg className={cx(variant)} />
+        <svg focusable={false} className={cx(variant)} />
```

Comment:
lets also recommend this pattern in our documentation for consumers using the custom glyph prop

Reply:
Done 322f801